app.py

```diff
@@ -44,17 +44,20 @@ def index():
 				TRAVIS_TAG = urlify(TRAVIS_TAG)#this will fix url issue
 				os.environ["TRAVIS_TAG"] = TRAVIS_TAG
 				os.environ["event_url"] = event_url
+				with open('travis_script_1.sh','rb') as f:
+					os.environ["TRAVIS_SCRIPT"] = str(base64.b64encode(f.read()))[1:]
```

Comment:
we can use it for sending wallpaper also as current system we host wallpaper due to which more than one user cannot use the webapp simultaneously